doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -323,6 +323,8 @@ Modem libraries
 
     * Current system time is attached to the ``location_datetime`` parameter of the location request response with Wi-Fi and cellular methods.
       The timestamp comes from the moment of scanning or neighbor measurements.
+    * Fixed errant dependency on modem_jwt library.
+      modem_jwt is now only required when :kconfig:`CONFIG_NRF_CLOUD_MQTT` is not enabled.
```

Comment:
```suggestion
    * Fixed errant dependency on the :ref:`lib_modem_jwt` library.
      The library is now only required when the :kconfig:`CONFIG_NRF_CLOUD_MQTT` option is not enabled.
```

Reply:
Docs builder is flatly refusing to accept the `:kconfig:` prefix on these line items, but accepting it just a couple of lines later. I am pretty flummoxed, do you have any idea what the cause here might be?

Reply:
Maybe related to Gerard Marull's email this morning:
•	I suggest rebasing all PRs, specially those containing documentation changes. Kconfig symbols are now referenced using :kconfig:option:`CONFIG_NAME`.

Reply:
That did the trick!